manifests/r/RoyalApps/RoyalTS/6.1.60512.0/RoyalApps.RoyalTS.installer.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: RoyalApps.RoyalTS
+PackageVersion: 6.1.60512.0
+MinimumOSVersion: 10.0.0.0

```

```suggestion
MinimumOSVersion: 10.0.0.0
Scope: machine
```
Package installs machine-wide (for all users)